hawkular-wildfly-agent-feature-pack/src/main/resources/modules/system/layers/base/org/hawkular/agent/main/module.xml

```diff
@@ -83,6 +83,7 @@
     <module name="org.jboss.vfs"/>
 
     <!-- other dependencies we need -->
+    <module name="javaee.api"/> <!-- bus-commons -->
```

Comment:
This is another reason why we can't have BasicMessage and related classes importing JMS classes - because there will be environments where that won't be available. The agent's use of bus-common is only BasicMessage and the JSON stuff. It doesn't even use the bus consumer/producer stuff.